### ISSUE TITLE
frictionlessdata markup?

### ISSUE THREAD

**Issue by evanwill** [CollectionBuilder/collectionbuilder-sa#8]
auto generated datapackage.json table schema, basically a data dictionary for the metadata, https://specs.frictionlessdata.io/data-package/

https://frictionlessdata.io/

data package creator, https://create.frictionlessdata.io/

**Comment by evanwill** [comment]
I think it might make sense to organize the data/ directory to match frictionless data-package spec, as if the "data/" is a data-package. Maybe move data/index.html out to data.html, and the internal used data (metadata.min and lunrstore.min ) out as well. Then the whole "data/" folder is a coherent chunk that is self-described by datapackage.json and could be imagined as something you might download complete (or somehow package as .zip)

this would involve change to the current standard location for the data/index.html page, however wouldn't change the data locations.

**Comment by evanwill** [comment]
I think the "_data" and "data" folder thing is a source of confusion. Another option could be to move /data/ to /export/ ? 
Then inside /export/ would be datapackage.json and a folder "data".

```
/export/datapackage.json 
/export/data/metadata.json
/export/data/metadata.csv
```

etc...

**Comment by evanwill** [comment]
group votes to put data package in `/assets/data/` and table page in `/data.html`

**Comment by evanwill** [comment]
in `data-reorg` branch: 

- [x] /data/index.html --> /data.html
- [x] lunr and table data --> /assets/js/
- [x] /data/ files --> /assets/data/
- [x] add /assets/data/datapackage.json

**Comment by evanwill** [comment]
closed by #10
